NSFileManager+TEAdditions.m

```diff
@@ -420,22 +468,35 @@ - (BOOL)copyOrMoveContentsFrom:(NSURL*)fromURL
                           move:(BOOL)move
                          error:(NSError**)error
 {
-    OSStatus err;
+    OSStatus err = noErr;
     NSError *ourErr;
-    FSRef fromRef, toRef;
+  
     NSURL *resultURL;
-    FSFileOperationClientContext context;
-    NSArray *subitems, *blacklist = $a(@".DS_Store", @".Trashes", @".fseventsd", @".Spotlight-V100",
-                                       @".com.apple.timemachine.supported", @".VolumeIcon.icns", @"Icon\r",
-                                       @".DocumentRevisions-V100", @".hotfiles.btree");
     BOOL success = YES;
+  NSArray *subitems, *blacklist = @[@".DS_Store", @".Trashes", @".fseventsd", @".Spotlight-V100",
```

Comment:
The indentation here is all messed up too, please make sure it's consistent everywhere